docs/source/app_developers_guide/kubernetes_test_network.rst

```diff
@@ -573,14 +573,14 @@ create and submit a batch of transactions containing the configuration change.
 
      .. code-block:: console
 
-        root@sawtooth-0# sawset proposal create --key /etc/sawtooth/keys/validator.priv \
+        root@sawtooth-0# sawset proposal create --key /root/.sawtooth/keys/my_key.priv \
```

Comment:
Yep. The commit that made the documentation incorrect is in the commit message.

https://github.com/hyperledger/sawtooth-core/pull/2128

Reply:
Hmmm.  This change makes other parts of the documentation incorrect or confusing (3+ notes and several steps in the App Dev and Sys Admin guides). However, this change makes the doc match the contents of the Kubernetes config file, which is the right thing to do.

I don't recommend adding another layer of explanation to any of the notes, because that would only increase the confusion. Instead, I propose a separate PR to update the other procedures and notes to use root's `my_key.priv` instead of the `validator.priv` key.

Reply:
For now, this change requires a change to two other steps in Step 7 (same file).